index.html

```diff
@@ -522,6 +509,20 @@ <h3 class="bannerbox__title">Can I help some other way?</h3>
                         </form>
                     </div>
                 </div>
+                <div class="bannerbox" id="righttovote">
+                    <h3 class="bannerbox__title">Voting support</h3>
+                    <div class="bannerbox__content">
+                        <a href="http://www.866ourvote.org/"><img class="align-right" src="res/img/election-protection.png" /></a>
+                        <p>The <a href="http://www.866ourvote.org/">Election Protection Coalition</a> works around the clock to advance and defend your right to vote.</p>
+                        <p>Report voting problems, and get voting relating info for your state.</p>
+                        <ul>
+                            <li>Visit <a href="http://www.866ourvote.org/">866ourvote.org</a></li>
+                            <li>Call <a href="tel:8666878683">866-OUR-VOTE</a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div>                   
+                </div>
```

Comment:
That's a clever solution, I wouldn't have thought of it. It's not ideal in terms of adding unnecessary markup, but if gets the job done 👍